networking.adoc

```diff
@@ -0,0 +1,1081 @@
+= Networking
+
+toc::[]
+
+
+## Configuring a custom PKI
+
+[source,terminal]
+----
+$ oc create -f user-ca-bundle.yaml
+----
+
+
+[source,terminal]
+----
+$ oc edit proxy/cluster
+----
+
+## Configuring the node port service range
+
+[source,terminal]
+----
+$ oc patch network.config.openshift.io cluster --type=merge -p \
+  '{
+    "spec":
+      { "serviceNodePortRange": "30000-<port>" }
+  }'
+----
+
+## Configuring PTP hardware
+
+[source,terminal]
+----
+$ oc create -f <filename> 
+----
+
+## Configuring ExternalIPs for services
+
+[source,terminal]
+----
+$ oc edit networks.config cluster
+----
+
+## Configuring ingress cluster traffic on AWS using a Network Load Balancer
+
+[source,terminal]
+----
+$ oc create -f ingresscontroller-aws-nlb.yaml
+----
+
+## Configuring ingress cluster traffic using an Ingress Controller
+
+[source,terminal]
+----
+$ oc new-project <project_name>
```

Comment:
I see a command coming before this in the docs which needs a declarative alternative:
```
$ oc adm policy add-cluster-role-to-user cluster-admin username
```

https://docs.openshift.com/container-platform/4.7/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.html#prerequisites